spec/support/rummager_helper.rb

```diff
@@ -0,0 +1,17 @@
+module RummagerHelper
+  def stub_rummager_linked_content_call
+    stub_request(:get, %r[https://rummager.test.gov.uk/.]).
```

Comment:
It would be better to use `Plek.find('rummager')` here to avoid hardcoding this domain etc.